Dockerfile.test-infra

```diff
@@ -1,42 +1,22 @@
-
-FROM jimschubert/swagger-codegen-cli:2.3.1 AS swagger
-
-# Generates bm-inventory-client
-RUN wget https://raw.githubusercontent.com/filanov/bm-inventory/master/swagger.yaml -O /swagger.yaml && \
-    sed -i '/pattern:/d' /swagger.yaml && \
-	echo '{"packageName" : "bm_inventory_client", "packageVersion": "1.0.0"}' > /config.json
-
-RUN /docker-entrypoint.sh generate --lang python --config /config.json --output /bm-inventory-client --input-spec /swagger.yaml
-
 FROM centos:8
-ENV container docker
 
-RUN yum -y install systemd make python3 virt-viewer virt-install libvirt-client git libvirt-devel gcc unzip wget jq && yum clean all; \
-(cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
-rm -f /lib/systemd/system/multi-user.target.wants/*;\
-rm -f /etc/systemd/system/*.wants/*;\
-rm -f /lib/systemd/system/local-fs.target.wants/*; \
-rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
-rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
-rm -f /lib/systemd/system/basic.target.wants/*;\
-rm -f /lib/systemd/system/anaconda.target.wants/*
+RUN yum -y install make python3 libvirt-client libvirt-devel gcc unzip wget curl git python3-devel && yum clean all;
 
 RUN wget `wget https://www.terraform.io/downloads.html -q -O - | grep -oP "(https://releases.hashicorp.com/terraform/.*linux_amd64\.zip)(?=\")" | head -n 1` && unzip terraform*.zip -d /usr/bin/ && rm -rf terraform*.zip
 
-RUN wget https://dl.google.com/go/go1.13.4.linux-amd64.tar.gz && tar -C /usr/local -xf go1.13.4.linux-amd64.tar.gz && rm -f go1.13.4.linux-amd64.tar.gz
+RUN curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.10.1/minikube-linux-amd64 \
+  && chmod +x minikube && mkdir -p /usr/local/bin/ && install minikube /usr/local/bin/
+RUN curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/bin/
+
 ENV PATH /usr/local/go/bin:$PATH
 RUN mkdir -p ~/.terraform.d/plugins
-RUN go get -v -u github.com/dmacvicar/terraform-provider-libvirt && cd ~/.terraform.d/plugins && go build -a -v github.com/dmacvicar/terraform-provider-libvirt && rm -rf ~/go
```

Comment:
Use [dockerfile best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)

```shell
curl -SL https://github.com/dmacvicar/terraform-provider-libvirt/releases/download/v0.6.2/terraform-provider-libvirt-0.6.2+git.1585292411.8cbe9ad0.Fedora_28.x86_64.tar.gz | tar -xJC ~/.terraform.d/plugins
```

Reply:
i removed the line, this is the best practice :)

Reply:
Are you kidding me? I commented on the removal but you added it back again horribly on line 13...